mobile/ios/ios-device.ts

```diff
@@ -304,6 +332,18 @@ export class IOSDevice implements Mobile.IIOSDevice {
 		}).future<void>()();
 	}
 
+	private startHouseArrestService(bundleId: string): void {
+		var func = () => {
+			var fdRef = ref.alloc("init");
+			var result = this.$mobileDevice.deviceStartHouseArrestService(this.devicePointer, this.$coreFoundation.createCFString(bundleId), null, fdRef);
+			var fd = fdRef.deref();
+
+			if(result !== 0) {
+				this.$errors.fail("AMDeviceStartHouseArrestService returned %s", result);
```

Comment:
If this error is visible to the users, we should guide them how to fix the underlying issue.